application/src/main/java/org/togetherjava/tjbot/features/basic/Starboard.java

```diff
@@ -1,14 +1,24 @@
 package org.togetherjava.tjbot.features.basic;
 
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.Permission;
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.entities.channel.concrete.TextChannel;
+import net.dv8tion.jda.api.entities.channel.middleman.GuildChannel;
 import net.dv8tion.jda.api.entities.channel.middleman.MessageChannel;
 import net.dv8tion.jda.api.events.message.react.MessageReactionAddEvent;
 import net.dv8tion.jda.api.hooks.ListenerAdapter;
+import net.dv8tion.jda.internal.utils.PermissionUtil;
 import org.jetbrains.annotations.NotNull;
 
 import org.togetherjava.tjbot.config.Config;
 import org.togetherjava.tjbot.config.StarboardConfig;
 import org.togetherjava.tjbot.features.EventReceiver;
 
+import java.awt.*;
```

Comment:
that import is "forbidden". please adjust ur intellij settings to disallow star-import on java.awt.

the reason is bc of java.awt.List, clashing with java.util.List

Reply:
Yeah I accidently imported that while importin `java.util.List`, ill remove that along with the other unused imports.